src/ducks/schema-selector.js

```diff
@@ -0,0 +1,24 @@
+import globallyResettableReducer from '../reducers/global-resettable';
+
+const SET_MOCKED_JSON_SCHEMA = 'SET_MOCKED_JSON_SCHEMA';
+
+export const setMockedJSONSchema = (payload) => ({
+  type: SET_MOCKED_JSON_SCHEMA,
+  payload
+});
+
+const INITIAL_SCHEMA_SELECTOR_STATE = {
+  schema: null
+};
+
+const schemaSelectorReducer = (state, action) => {
+  const { payload, type } = action;
+  if (type === SET_MOCKED_JSON_SCHEMA) {
+    return {
+      schema: payload
+    };
+  }
+  return state;
+};
+
+export default globallyResettableReducer(schemaSelectorReducer, INITIAL_SCHEMA_SELECTOR_STATE);
```

Comment:
I think it would be a good idea to persist this reducer with `redux-persist` so we don't have to choose the schema everytime we reload the page.